src/iOS/toga_iOS/libs/__init__.py

```diff
@@ -1,3 +1,4 @@
 from .core_graphics import *
```

Comment:
At column 1: (F403) 'from .core_graphics import *' used; unable to detect undefined names